docs/user/supported_devices.rst

```diff
@@ -91,6 +91,7 @@ ath79-generic
 
   - UniFi AC Mesh
   - UniFi AP
+  - UniFi AP AC Lite
```

Comment:
For consistency with the other UniFi AC variants, this should be "UniFi AC Lite" without the "AP".

Reply:
resolved in d3a2c657f912b1d7354c962fc91bcc5ba437f739